rgw/v2/tests/s3_swift/test_compression_encryption.py

```diff
@@ -0,0 +1,209 @@
+"""
+test_compression_encryption.py - using boto
+
+Usage: test_compression_encryption.py -c <input_yaml>
+<input_yaml>
+    test_compression_encryption.yaml
```

Comment:
@viduship , please add both configs in this PR

Reply:
thanks @hmaheswa , I have updated the second yaml as well